.ci/examples/job_matrix_actions.yaml

```diff
@@ -13,10 +13,20 @@ volumes:
 runs_on_dockers:
   - {name: 'blackduck', url: 'harbor.mellanox.com/toolbox/ngci-centos:latest', category: 'tool', arch: 'x86_64'}
   - {name: 'coverity', url: 'harbor.mellanox.com/swx-storage/ci-demo/x86_64/centos7-7:latest', category: 'tool', arch: 'x86_64'}
+  - {name: 'builder', url: 'harbor.mellanox.com/swx-storage/ci-demo/x86_64/centos7-7:latest', arch: 'x86_64'}
 
+credentials:
+  - {credentialsId: '341392c9-cb1a-4d5d-1ca4-6ed43ba0r02c', usernameVariable: 'NEXUS_USER', passwordVariable: 'NEXUS_PASS'}
 
 steps:
 
+  - name: Nexus upload rpm
+    resource: actions/nexus.py
+    containerSelector: "{name: 'builder'}"
+    run: |
+      ./build_package.sh
+      .ci/actions/nexus.py yum -u http://swx-repos.mtr.labs.mlnx:8081/ -U $NEXUS_USER -P $NEXUS_PASS -a upload -n test_yum_repo --file /tmp/mlnx-libsnap-1.0.1-16.mlnx.x86_64.rpm --upload_path 7/x86_64/
```

Comment:
is there way to make nexus url as variable as well? (maybe as part of credentials env var) ?

Reply:
Yes, sure
I nvmx project I pass it using jjb as job parameter - https://github.com/Mellanox/nvmx/blob/main/.ci/proj_jjb.yaml#L39

Reply:
Added as env var to job_matrix_actions.yaml